openapi3.yaml

```diff
@@ -61,12 +147,37 @@ components:
       properties:
         message:
           type: string
-    resource:
+    requestBodyForFilter:
```

Comment:
Schemas should have meaningful names, something to explain what they stand for.

Reply:
is `filterAndPaginationToCatalog` make more sence?